src/ui/component/Learning.html

```diff
@@ -1,47 +1,45 @@
-                </a>
-            {{else}}
-                <a href="/courses/{{course.uid}}" class="learning__title">
-                    {{course.title}}
-                </a>
-            {{/if}}
+<div class="resource__item">
+    <div class="resource__main">
+        {{#if course.type === 'link'}}
+            <a href="/courses/{{course.uid}}" class="resource__title" target="_blank">
+                {{course.title}}
+            </a>
+        {{else}}
+            <a href="/courses/{{course.uid}}" class="learning__title">
+                {{course.title}}
+            </a>
+        {{/if}}
+        <span class="resource__action">
             {{#if course.state}}
                 {{#if course.type === 'face-to-face'}}
                     <a href="/book/{{course.uid}}/cancel">Cancel</a>
                 {{else}}
-                    {{#if course.type === 'elearning'}}
-                        <a href="/courses/{{course.uid}}/do">Resume</a>
-                    {{else}}
-                        <a href="/courses/{{course.uid}}">Resume</a>
-                    {{/if}}
+                <a href="/courses/{{course.uid}}/{{course.type === 'elearning' ? 'do' : ''}}">Resume</a>
                     {{#if !course.isRequired(signedInUser)}}
                         <a href="/courses/{{course.uid}}/delete">Delete</a>
                     {{/if}}
                 {{/if}}
             {{/if}}
-        </div>
-
-        <span class="learning__progress">{{i18n(course.state || 'not started')}}</span>
-
-        <div class="column-one-third learning__information">
-            <p class="learning__information-desc">
-                {{#if course.requiredBy}}
-                    Required by {{formatDate(course.nextRequiredBy())}}
-                {{/if}}
-                {{#if course.selectedDate}}
-                    Registered for {{formatDate(course.selectedDate)}}
-                {{/if}}
-            </p>
+        </span>
+    </div>
+    <div class="resource__info">
+        <span>
+            {{#if course.requiredBy && course.type === ('elearning' || 'face-to-face') }}
+                Required by {{formatDate(course.nextRequiredBy())}}
+            {{/if}}
+            {{#if course.selectedDate}}
+                Registered for {{formatDate(course.selectedDate)}}
+            {{/if}}
+        </span>
 
-            <ul class="learning__information-list">
-                <li>{{i18n(course.type)}}</li>
-                <li>{{course.duration}}</li>
-            </ul>
-        </div>
+    </div>
+    <div class="resource__duration">
+        <span>{{course.type === ('elearning' || 'face-to-face') ? '' : course.type + ', '}}{{course.duration}}</span>
+    </div>
+    <div class="resource__status">
+        <span class="badge badge--info badge--smaller">{{i18n(course.state || 'not started')}}</span>
+        {{#if course.requiredBy && course.type !== ('elearning' || 'face-to-face') }}
+            <span class="meta">21 Aug 2018</span>
```

Comment:
What's this date meant to be? Date started, booked, progressed..? We'll need to add to the learner record.

Reply:
thanks for the catch, it's just course.requiredBy for non elearning/face to face